module/zfs/dnode.c

```diff
@@ -86,6 +86,19 @@ int zfs_default_ibs = DN_MAX_INDBLKSHIFT;
 static kmem_cbrc_t dnode_move(void *, void *, size_t, void *);
 #endif /* _KERNEL */
 
+static inline char *
```

Comment:
```suggestion
static char *
```

No need for the `inline` keyword hint here.  The function is `static` with a single caller so I'd be shocked if the compiler didn't inline it.  As for the instance in `metslab.c` I'd suggest dropping it there too, but I doubt it matters much.

Reply:
Fixed. The metaslab one is exported now for re-use in another TU.